acceptance/README.md

```diff
@@ -0,0 +1,26 @@
+# Acceptance Tests for chef
+
+This directory contains acceptance tests for Chef Client. These tests run in Chef's release infrastructure. They are also runnable from CLI.
+
+```
+$ cd acceptance
+$ bundle install
+$ bundle exec chef-acceptance test gem_versions
+```
+
+Note:
+
+* If you are using Chef Development Kit's ruby environment make sure you set this environment variable:
+  `export APPBUNDLER_ALLOW_RVM=true`
+* You need to have AWS configuration to be able to create VMs in EC2.
```

Comment:
Can you link it like `[AWS configuration](https://github.com/test-kitchen/kitchen-ec2/#authenticating-with-aws)` plz?